app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,24 @@
+class RatingsController < ApplicationController
+  expose :rating
+  before_action :authenticate_user!
+  expose_decorated :article
+  expose :rating, build: ->(rating_params) { Rating.new(rating_params) }
```

Comment:
I'm not sure that we should pass `params[:rating][:rate]` for build 
```suggestion
  expose :rating, build: ->(rating_params) { current_user.comments.new(article: article) }
```